packages/wonder-blocks-toolbar/src/components/__docs__/toolbar.stories.js

```diff
@@ -162,6 +162,28 @@ HeaderOverflowText.parameters = {
     },
 };
 
+/**
+ * Flexible toolbars.
+ */
+export const Responsive: StoryComponentType = Template.bind({});
+
+Responsive.args = {
+    leftContent: leftContentMappings.hintButton,
+    rightContent: rightContentMappings.responsive,
```

Comment:
No action required, but I think once unfortunate thing about reusing args like this (which is a great reuse of variables!) is that it makes the code harder to read in the "Show code" section of a story.

I was just experimenting to see how someone would use this story to figure out how to make their own toolbar responsive, and it's a little hard to tell when all the code is on one line 😕 

<img width="1063" alt="Screen Shot 2022-11-10 at 2 00 32 PM" src="https://user-images.githubusercontent.com/13231763/201214935-b0d40807-03df-44fe-9098-8a6191567079.png">